### PR TITLE
Update Set-MpPreference.md

### PR DESCRIPTION
**Pull request by marysiakam9889**
Adding new update control commands for SDP scenario

### CODE REVIEW

**Review by pahuijbr — APPROVED** [review]
Please approve this PR, marysiakam9889 is also MSFT internal.

File: docset/winserver2019-ps/defender/Set-MpPreference.md [hunk 1]
```diff
@@ -137,6 +137,30 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DefinitionUpdatesChannel
+Enable this policy to specify when devices receive daily Microsoft Defender definition updates during the daily gradual rollout. 
+
+Current Channel (Staged): Devices will be offered updates after the release cycle. Suggested to apply to a small, representative part of production population (~10%)
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:143]:
```suggestion
Current Channel (Staged): Devices will be offered updates after the release cycle. Suggested to apply to a small, representative part of production population (~10%).
```

File: docset/winserver2019-ps/defender/Set-MpPreference.md [hunk 2]
```diff
@@ -248,6 +272,28 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisableGradualRelease
+
+Enable this policy to disable gradual rollout of monthly and daily Defender updates. 
+
+Devices will be offered all Defender updates after the gradual release cycle completes. Best for datacenter machines that only receive limited updates. 
+
+Note: This setting applies to both monthly as well as daily Defender updates and will override any previously configured channel selections for platform and engine updates.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:281]:
```suggestion
> [!NOTE]
> This setting applies to both monthly as well as daily Defender updates and will override any previously configured channel selections for platform and engine updates. 
```

File: docset/winserver2019-ps/defender/Set-MpPreference.md [hunk 3]
```diff
@@ -137,6 +137,30 @@ Accept pipeline input: False
...
+Aliases:
+Accepted values: Staged|Broad|Not Configured
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:163]:
```suggestion
```

File: docset/winserver2019-ps/defender/Set-MpPreference.md [hunk 4]
```diff
@@ -585,6 +661,36 @@ Accept pipeline input: False
...
+Accepted values: Beta|Preview|Staged|Broad|NotConfigured 
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+
+
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:693]:
```suggestion
```

File: docset/winserver2019-ps/defender/Set-MpPreference.md [hunk 5]
```diff
@@ -585,6 +661,36 @@ Accept pipeline input: False
...
+Aliases:
+Accepted values: Beta|Preview|Staged|Broad|NotConfigured 
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:692]:
```suggestion
```

File: docset/winserver2019-ps/defender/Set-MpPreference.md (hunk 4 above)
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:693]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @marysiakam9889, please commit the few minor suggested changes. Many thanks.

File: docset/winserver2019-ps/defender/Set-MpPreference.md [hunk 6]
```diff
@@ -137,6 +137,30 @@ Accept pipeline input: False
...
+Current Channel (Broad): Devices will be offered updates only after the gradual release cycle completes. Suggested to apply to a broad set of devices in your production population (~10-100%). 
+
+If you disable or do not configure this policy, the device will stay up to date automatically during the daily release cycle. Suitable for most devices. 
+
+Supported OS versions: Windows 10 
+
+```yaml
+Type: UpdatesChannelType
+Parameter Sets: (All)
+Aliases:
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:155]:
```suggestion
Aliases:

```

File: docset/winserver2019-ps/defender/Set-MpPreference.md [hunk 7]
```diff
@@ -248,6 +272,28 @@ Accept pipeline input: False
...
+
+Note: This setting applies to both monthly as well as daily Defender updates and will override any previously configured channel selections for platform and engine updates. 
+
+If you disable or do not configure this policy, the device will remain in Current Channel (Default) unless specified otherwise in specific channels for platform and engine updates. Stay up to date automatically during the gradual release cycle. Suitable for most devices. 
+
+Supported OS versions: Windows 10 
+
+```yaml
+Type: Boolean
+Aliases:
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:290]:
```suggestion
Aliases:

```

File: docset/winserver2019-ps/defender/Set-MpPreference.md [hunk 8]
```diff
@@ -410,6 +457,35 @@ Accept pipeline input: False
...
+If you disable or do not configure this policy, the device will stay up to date automatically during the gradual release cycle. Suitable for most devices. 
+
+Supported OS versions: Windows 10 
+
+```yaml
+Type: UpdatesChannelType
+Parameter Sets: (All)
+Aliases:
+Accepted values:Beta|Preview|Staged|Broad|NotConfigured 
+
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:482]:
```suggestion
Aliases:

Accepted values:Beta|Preview|Staged|Broad|NotConfigured 
```

File: docset/winserver2019-ps/defender/Set-MpPreference.md [hunk 9]
```diff
@@ -585,6 +661,36 @@ Accept pipeline input: False
...
+Current Channel (Broad): Devices will be offered updates only after the gradual release cycle completes. Suggested to apply to a broad set of devices in your production population (~10-100%). 
+
+If you disable or do not configure this policy, the device will stay up to date automatically during the gradual release cycle. Suitable for most devices. 
+
+Supported OS versions: Windows 10 
+
+```yaml
+Type: UpdatesChannelType
+Parameter Sets: (All)
+Aliases:
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:684]:
```suggestion
Aliases:

```

**Pull request merged**